project/Doc.scala

```diff
@@ -99,6 +105,9 @@ object UnidocRoot extends AutoPlugin {
     scalacOptions in (ScalaUnidoc, unidoc) ++= Seq("-skip-packages", "com.lightbend.lagom.internal"),
     javacOptions in doc := Seq(
       "-windowtitle", "Lagom Services API",
+      "-link", "https://docs.oracle.com/javase/8/docs/api/",
+      "-linkoffline", "https://doc.akka.io/japi/akka/current/", packageListLocation(baseDirectory.value, "akka").getAbsolutePath,
+      "-linkoffline", "https://www.playframework.com/documentation/latest/api/java/", packageListLocation(baseDirectory.value, "play").getAbsolutePath,
```

Comment:
Using the `-linkoffline` alternative works but requires maintaining the `package-list` files.

Also, this will never work x-referencing play because in parts of our code we reference `play.api.*` which is not in Play's javadocs. This create broken links in our docs. 🤦‍♂️

Reply:
Akka publishes `package-list` files (apparently) under https://doc.akka.io/japi/akka/current/package-list. So you should be able to use `-link https://doc.akka.io/japi/akka/current/`